cmd/kots/cli/discover.go

```diff
@@ -0,0 +1,201 @@
+package cli
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+func discover() (string, string, string) {
+	var dist string
+	var version string
+
+	var lsbDist string
+	var distVersion string
+	var distVersionMajor string
+	var majorVersion string
+
+	// Find the Linux Standard Distribution
+	if _, err := os.Stat("/etc/centos-release"); err == nil {
+		// this is for centos
+		distCmd := fmt.Sprintf("%s", "cat /etc/os-centos-release | cut -d\" \" -f1")
+		dist, _ = getBashData("/etc/os-centos-release", distCmd)
```

Comment:
Are you going to add error checking later?

Reply:
all these code are gone. using library